mozilla_django_oidc/views.py

```diff
@@ -60,14 +69,18 @@ def __init__(self, *args, **kwargs):
 
     def get(self, request):
         """OIDC client authentication initialization HTTP endpoint"""
+        state = get_random_string(import_from_settings('OIDC_STATE_SIZE', 32))
```

Comment:
I suggest we keep it this way since its outside of this PRs scope (`state` is already added in the request, we are just checking against the session) and open an issue for that.